R/zzz.R

```diff
@@ -0,0 +1,24 @@
+.onLoad <- function(libname, pkgname) {
+
+  .auth <<- gargle::init_AuthState(
+    package     = "googledrive",
+    auth_active = TRUE
+  )
+
+  if (identical(Sys.getenv("IN_PKGDOWN"), "true")) {
+    tryCatch(
+      drive_auth_docs(),
+      googledrive_auth_internal_error = function(e) NULL
+    )
+  }
+
+  if (is_installed("dplyr", version = "1.0.0")) {
+    vctrs::s3_register(
+      "dplyr::dplyr_reconstruct",
+      "dribble",
+      method = dribble_maybe_reconstruct
+    )
+  }
```

Comment:
I have to do this since dplyr is only in Suggests. I've got this condition, because I'm willing to only support dplyr compatibility from v1.0.0 on. And this use of `version` is, in fact, why I depend on dev rlang in this PR (but I will need this for separate rlang+cli reasons anyway).

Reply:
That's fine, its very similar to what we do in tune, since at the time we had to conditionally register depending on the dplyr version that was being used (even though it was in Imports) https://github.com/tidymodels/tune/blob/b0e83a776df5779ee325938b217ce7a5192b715f/R/zzz.R#L78

I have a faint memory of `is_installed()` not working correctly inside `.onLoad()`, but can't remember exactly where that came up. But if it isn't bothering you, I wouldn't worry about it.